.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -29,7 +29,10 @@ jobs:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'linux_x64_clang')
     runs-on: azure-linux-scale
-    container: ghcr.io/iree-org/cpubuilder_ubuntu_jammy_x86_64@sha256:ba10d33ef502d8b33aaa43800e640a6866983ef353b73fda2ed6595c4275e066
+    container:
+      # blobfuse requires the privileged flag
+      image: ghcr.io/iree-org/cpubuilder_ubuntu_jammy_x86_64:main
```

Comment:
Thoughts on this? using the hash I couldn't update the container, but do we always want to use the most up to date

Reply:
My concern with using `:main` is that if we push a bad update, workflows will be immediately affected by it. I'd rather pin to a version and then choose to rollout the update intentionally (with checks running on the update).

Reply:
ok ill find a hash of the new version